editor/src/components/canvas/controls/select-mode/control-common.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import { CSSNumber, CSSNumberUnit, printCSSNumber } from '../../../inspector/common/css-utils'
+
+export interface CSSNumberWithRenderedValue {
+  value: CSSNumber
+  renderedValuePx: number
+}
+
+function valueWithUnitAppropriatePrecision(unit: CSSNumberUnit | null, value: number): number {
+  if (unit === 'em') {
+    return Math.floor(value * 10) / 10
+  }
+  return Math.floor(value)
```

Comment:
Should this be rounding rather than flooring? If so, we have this function in `math-utils.ts` for that: <https://github.com/concrete-utopia/utopia/blob/8288f0d9f93a0fea10430af0599f1d049e29fb52/editor/src/core/shared/math-utils.ts#L790>

Reply:
thanks for the suggestion! i replaced the custom calculation with `roundTo`